app/Notifications/RequestAssetCancelation.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Setting;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Messages\SlackMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class RequestAssetCancelationNotification extends Notification
+{
+    use Queueable;
```

Comment:
Does this require people to use a Queue of some sort, or will it "just work" without one? If so, then that's fine.

Reply:
No queue required.